fe/fe-core/src/main/java/com/starrocks/alter/AlterJobMgr.java

```diff
@@ -998,6 +1005,7 @@ public void processAlterView(AlterViewStmt stmt, ConnectContext ctx) throws User
             }
             view.setNewFullSchema(newFullSchema);
 
+            LocalMetastore.inactiveRelatedMaterializedView(db, view, String.format("base view %s changed", viewName));
```

Comment:
Can we active the mv later after view has changed ?

Reply:
yes